src/main/java/frc/robot/subsystems/vision/Limelight.java

```diff
@@ -56,6 +56,15 @@ public Pipeline getPipeline() {
         return Pipeline.fromIndex((int) getpipe.get());
     }
 
+    public void togglePipeline() {
```

Comment:
Consider having this method returning a command, and then you won't need to create a command in `RobotContainer`.